admin/src/helpers/agent.js

```diff
@@ -12,14 +12,23 @@ export const agent = {
     });
   },
 
-  updateProduct: (productId, productData, token) => {
-    return fetch(`${baseUrl}/users/products/${productId}`, {
+  editProduct: (productId, productData, token) => {
+    return fetch(`${baseUrl}/products/${productId}`, {
       method: "get",
```

Comment:
:x: `get`
:white_check_mark: `put`
Since `editProduct` required `put` request to work